flake.nix

```diff
@@ -75,5 +82,29 @@
             buildUnisonFromTranscript = buildUnisonFromTranscript pkgs;
           };
       };
+
+      homeConfigurations = builtins.listToAttrs (map (system: {
+        name = "${system}-example";
```

Comment:
Minor: should this be something more like `"unison-home-manager-example-${system}"`?

Reply:
I don’t think so, because in practice it would be used like
```
home-manager build --flake github:ceedubs/unison-nix#x86_64-linux-example
```
or
```
nix build github:ceedubs/unison-nix#homeConfigurations.x86_64-linux-example.activationPackage
```
so “unison” and “home*” are apparent from the context in both cases.

I do think `example-${system}` looks nicer, but I use `${system}-example` to parallel the structure of other outputs, like `packages.${system}.ucm`.

But I’m not tied to any of the naming.

Reply:
Ohhh I didn't look carefully enough. I was thinking that this was the `pname` of a derivation. This is the key for the flake output. :+1: